components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -606,6 +606,28 @@
   width: 48%;
   float: right;
 }
+.menu-box .field-deceasedAge {
+  width: 47%;
+  float: left;
+}
+.menu-box .field-deceasedCause {
+  width: 48%;
+  float: right;
+}
+.menu-box .field-deceasedAge {
+  padding-top: 0;
+  padding-bottom: 0.8em;
+}
+.menu-box .field-deceasedCause {
+  padding-top: 0;
+  padding-bottom: 0.8em;
+}
```

Comment:
that was my original intention to merge them, but @allasm disagreed so I followed his lead on this